index.html

```diff
@@ -410,6 +410,26 @@ <h2>Privacy and Security</h2>
     mark or measure recorded by any script included on the page can be read by
     any other script running on the same page, regardless of origin.</p>
   </section>
+  <section id="recommended-mark-names" class="informative">
```

Comment:
Quick drive-by comment: I think I'd move this up in the document, before the Privacy and Security sections, which are traditionally right before the appendices.

Reply:
Updated.